React/Views/RCTScrollView.m

```diff
@@ -358,6 +359,23 @@ - (UIView *)hitTest:(CGPoint)point withEvent:(UIEvent *)event
   return [super hitTest:point withEvent:event];
 }
 
+- (BOOL)gestureRecognizerShouldBegin:(UIGestureRecognizer *)gestureRecognizer {
+  // ScrollView enables pinch gesture late in its lifecycle. So simply setting it
+  // in the setter gets overriden when the view loads. Revoke & disable gesture
+  // at first use.
+  if (!_pinchEnabled && [gestureRecognizer isKindOfClass:[UIPinchGestureRecognizer class]]) {
```

Comment:
@dlowder-salesforce It's a subclass of ScrollView, not ScrollViewController. Lifecycle methods aren't available.